proposal.rst

```diff
@@ -273,11 +399,31 @@ Use of Existing Code
 **If you already know about existing code you plan to incorporate or libraries
 you plan to use, please give details.**
 
-FILLME
+I have cited the code-base references (folders, git-repo) above as they were relevant to the context. However still for my background research I went through all documentation on Xapian docs (from xapian-docsprint), Omega (Overview and few other ones), Letor(the ML workflow), Yahoo and Letor research paper(to understand ML theory including dataset-creation , feature-selection)
+
+Still, 
+
+- weight-scheme algorithm definition : xapian-core/weight
+- weight-scheme algorithm calling API : xapian-core/bin/*
+- weight-scheme algorithm workflow example : xapian-core/tests/api_backend.cc and others.
+- weight-scheme evaluation : xapian-evalution
+
+- letor algorithm definition : xapian-letor/scorer, xapian-letor/feature
+- letor algorithm calling API : xapian-letor/bin/xapian-train and xapian-letor/bin/xapian-rank.cc
+- letor algorithm workflow example : xapian-letor/tests/api_letor.cc 
+- letor algorithm evaluation : xapian-letor/ranker. 
+- letor docs : xapian-letor/docs
+
+- Omega templates : xapian-applications/omega/templates..
+- Stemming data : xapian-data/stemming
+
+
+
```

Comment:
This isn't asking about code that's already in Xapian - see comment in the template below for the motivation for this question.

Reply:
I see that. I have updated my proposal with the external libraries tool I am looking to incoporate. Thanks for the mention.